.github/workflows/tpv.yml

```diff
@@ -33,32 +33,22 @@ jobs:
           git submodule update --init --recursive --remote --checkout
 
       - name: Set up Python 3.
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v6
         with:
-          python-version: "3.11"
-          cache: "pip"
+          python-version: "3.13"
+
+      - name: Install uv.
+        uses: astral-sh/setup-uv@v7
```

Comment:
I don't know if we can actually.

https://docs.astral.sh/uv/guides/integration/github/#caching